CHANGELOG.md

```diff
@@ -1,5 +1,13 @@
 # Changelog
 
+## Bolt NEXT
+
```

Comment:
Given we had merged the 1.36.0 changelog finalization last week before we found the ship stopper bug, Bolt Next *is* 1.36.0. You can either add it under that section, or change 1.36.0 to Bolt Next.